composer.json

```diff
@@ -57,29 +57,17 @@
             "homepage": "http://dynasource.eu"
         }
     ],
+    "minimum-stability": "dev",
     "require": {
-        "yiisoft/yii-base-web": "^3.0"
     },
     "require-dev": {
-        "yiisoft/yii-debug": "^3.0",
-        "yiisoft/yii-gii": "^3.0",
-        "phpunit/phpunit": "^7.0"
```

Comment:
why remove these?

Reply:
Because `yii-debug` is only useful when you use a web (and/or API) base. Similar about `yii-gii` although you can use it via web and CLI you might not need it if you create an API only project.